### ISSUE TITLE
Python 3 is not accessible with NVDA

### ISSUE THREAD

**Issue by jacobchawala** [nvaccess/nvda#9231]
I ask myself why Python 3 is not accessible with NVDA while Python 3 is the main program used to build, design, and develop NVDA. this is the high time to make it accessible with NVDA. for instance, i was writing a code on the python's editor, nothing was read by pressing any of my keyboard arrows! i went to the console to check its results, nothing was read by NVDA! so this first version of NVDA should be able to access Python 3 to help programmers with visual impairment to use Python 3 without using other external applications. to start with, at least, three things should be accessible with NVDA: its editor, console, and a debugger. Luckily, its menu bar is accessible.
I am using NVDA 2018.4.1. I have installed Python 3 64 bits. since the python's editor is not accessible, i use notepad++ to code.
system Information
Machine name: DESKTOP-R0NVNF4
               Machine Id: {F1EFE585-AA4A-485E-9FE9-4DC0FCCDACBF}
         Operating System: Windows 10 Pro 64-bit (10.0, Build 18323) (18323.19h1_release.190119-1759)
                 Language: English (Regional Setting: English)
      System Manufacturer: TOSHIBA
             System Model: Satellite C850-C035
                     BIOS: InsydeH2O Version 03.72.306.50 (type: BIOS)
                Processor: Intel(R) Core(TM) i3-2348M CPU @ 2.30GHz (4 CPUs), ~2.3GHz
                   Memory: 8192MB RAM

**Comment by lukaszgo1** [comment]
Duplicate of #3812. The referenced issue was closed as cantfix, and I am afraid we have to do the same with this one.

**Comment by ehollig** [comment]
Duplicate of #3812
Yes, unfortunately, Closing as cantfix. Feel free to reopen if you disagree. CC @michaelDCurran @jcsteh @leonardder @josephsl
